db/migrate/20211001131132_add_sub_heading_to_coronavirus_sub_section.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubHeadingToCoronavirusSubSection < ActiveRecord::Migration[6.1]
+  def change
+    add_column :coronavirus_sub_sections, :sub_heading, :string, limit: 255
```

Comment:
The setting of limit: 255 doesn't actually do anything here as I recall. MySQL just defaults to 255 chars so you can drop that from the arguments.

Reply:
got myself into a right git hole with that one, no idea how! At any rate it's gone now